docs/contracts-intro.md

```diff
@@ -2,11 +2,181 @@
 
 ---8<-- 'includes/data-builder-danger-header.md'
 
-This page will introduce OpenSAFELY Contracts, explaining:
+## OpenSAFELY Contracts are a clinical data specification
 
-* what they are
-* the process by which new contracts could be proposed
-* where [details of existing contracts will be
-  found](contracts-reference.md)
+*OpenSAFELY Contracts* are an important OpenSAFELY concept that is
+introduced with Data Builder.
+
+This page describes:
+
+* what OpenSAFELY Contracts are
+* how OpenSAFELY Contracts relate to dataset definitions
+
+Each OpenSAFELY Contract is a formal specification, [described in
+code](https://github.com/opensafely-core/databuilder/blob/main/databuilder/contracts/contracts.py),
```

Comment:
In fact that module is obsolete. The real contracts will be namespaced, for example the `universal` module. I suggest that you link to https://github.com/opensafely-core/databuilder/blob/main/databuilder/contracts which will have several modules and subfolders in it.